README.md

```diff
@@ -133,12 +133,122 @@ If you have all of your components in the folder `./comps` then you would call `
 
 Every folder directly under `./comps` would get processed and the build tools and handed off to rollup.
 
+## Locale files
+
+Locale files are JSON files that contains locale specific translations of strings you use in your application. By default the files are all placed in the `locales` folder and are names `strings_??.json` where the `??` is the 2 letter locale such as `en`, `fr`, or `ja`.
+
+The file `./locales/strings_en.json` will contain the English version of the strings. The file `./locales/strings_fr.json` contains the French version of the strings. You can use things like `en-US` for US english and `fr-CA` for Canadian French. The two letter [ISO 639-1](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) code and a subset of the [BCP-47](http://www.ietf.org/rfc/bcp/bcp47.txt) codes are supported.
+
+Currently we only support the 2 letter ISO 639-1 code and this ISO 639-1 code followed by a dash `"-"` and then the 2 letter ISO-3166 country code. Like `en`, `fr`, `en-US`, `en-GB`, `fr-CA` and `fr-FR`. **Case is ignored.**
+
+> [ISO-3166 Country Codes and ISO-639 Language Codes](https://docs.oracle.com/cd/E13214_01/wli/docs92/xref/xqisocodes.html)
+
+These locale files are combined into the file `./_compiled/locales.mjs` and you access them by importing that file and then calling the default function to get the set of locale strings for the specified locale.
+
+```JS
+import locales from "./_compiled/locales.mjs";
+const lang = locales('en');
+```
+
+> If no locale key is provided in the `locales()` function then the default locale object is returned. 
+
+## Templates
+
+Template files tend to be HTML files but can be any kind of file. The templates are embedded into a template literal inside of the `template.mjs` file.
+
+The template file `person.html`:
+
+```html
+<div class="first">
+  <div>Name: <span class="name">Frank N Stein</span></div>
+</div>
+```
+
+Would before the template:
```

Comment:
It seems like `before` was supposed to be `return` or `provide` or some other verb.

Reply:
Fixed